scripts/ci/github_actions/driver/wait_for_k8s_ready.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -xe
+set +o pipefail
+
+is_pod_ready (){
+  pods=$@
+  are_all_pods_ready=false
+  for pod in $pods; do
+    running_containers_count=`echo $pod | awk -F / '{print$1}'`
+    total_containers_count=`echo $pod | awk -F / '{print$2}'`
+    if [ $running_containers_count != $total_containers_count ]; then
+      are_all_pods_ready=true
+      break
+    fi
+  done
+  echo $are_all_pods_ready
+}
+
+is_kubernetes_cluster_ready (){
+  pods=`kubectl get pods -A | awk '{print$3}' | grep -iv ready`
+  all_containers_are_running=false
+  are_all_pods_ready=false
```

Comment:
Why do we need this line if we override the value in the line below

Reply:
done